### PR TITLE
Use versioned target population

### PR DESCRIPTION
**Pull request by patriciagarcia**
Connects #20 

- Updates the service to use the new versioned target population field
- Makes sure that if no version is provided for any of the factors, version 1 is used by default

### CODE REVIEW

**Comment by patriciagarcia** [comment]
Semantic release failing, it was working for me locally but I made the mistake of removing my `node_modules` folder to be able to replicate the error. Now I can see the error but I have no idea how to fix it ;-). 

Any clue https://travis-ci.org/fielded/angular-nav-thresholds?

**Comment by patriciagarcia** [comment]
Uh, actually what fails is `rollup -c` and it creates a weir file called `-c` that contains `undefined`

```
> npm run -s build && karma start
resolve failed:  { [Error: Cannot find module 'babel-runtime'] code: 'MODULE_NOT_FOUND' } babel-runtime
resolve failed:  { [Error: Cannot find module '@semantic-release'] code: 'MODULE_NOT_FOUND' } @semantic-release
Aborting commit due to empty commit message.
```

**Comment by tlvince** [comment]
Pinning didn't work, but can reproduce locally.

**Comment by tlvince** [comment]
Error is due to some publishing mess in https://github.com/rollup/babel-preset-es2015-rollup. Either way, that package is deprecated. Will push a fix direct to the branch here.

**Review by tlvince — APPROVED** [review]

File: src/thresholds.service.js [hunk 1]
```diff
@@ -40,40 +40,73 @@ const getFactor = (location, versions, version) => {
 }
 
 const getFactors = (stockCount, location, options) => {
-  if (!(location.allocations && location.allocations.length)) {
-    return {}
-  }
+  // centralized for whenever we implement #16
+  const somethingIsWrong = () => undefined
```
Comment (tlvince) [inline src/thresholds.service.js:44]: 😄

**Comment by patriciagarcia** [comment]
@tlvince, updated to take into account the proper target population field naming and to make it backwards compatible with non versioned TPs

File: src/thresholds.service.js (hunk 1 above)
Comment (patriciagarcia) [reply src/thresholds.service.js:44]: 😂  so descriptive!

**Pull request merged**